Sources/Castor/Castor.docc/Articles/playback/playback-article.md

```diff
@@ -9,4 +9,128 @@ Play audio and video content with ease.
 
 ## Overview
 
-TBD
+Use a ``CastPlayer`` to manage playback of one or more media assets sequentially, and receive automatic updates about playback state changes.
+
+> Important: ``CastPlayer`` cannot be instantiated manually, it must be retrieved from the ``Cast`` object.
+
+## Load content
+
+A content that can be played in the ``Cast`` universe is called a ``CastAsset``. Assets can be represented in three forms:
+
+- **Entity**: It is a deep link URL that represents a single media item and that your receiver should be able to handle.
+- **Identifier**: A unique ID for that represents a single media item and that your receiver should be able to handle.
+- **URL**: The direct URL of the content. This asset type may require additional configuration, which can be provided via ``CastAssetURLConfiguration``.
+
+> Warning: Playing a media URL directly can sometimes be tricky. For example, if you try to play an HLS URL directly, the Cast receiver might not respond correctly. This is because the receiver often requires additional information, such as the MIME type and the type of audio and video segments, to handle the stream properly. You can provide this information using the ``CastAsset/url(_:configuration:metadata:customData:)-6xskn`` initializer along with a ``CastAssetURLConfiguration``.
+
+You can load one or several items from media assets. Since ``CastPlayer`` conforms to [`ObservableObject`](https://developer.apple.com/documentation/combine/observableobject) and cannot be instantiate manually, it should be stored as a [`ObservedObject`](https://developer.apple.com/documentation/swiftui/observedobject) within a SwiftUI view. This ensures the player's lifecycle aligns with the view and that UI updates automatically reflect changes in playback state.
```

Comment:
```suggestion
You can load one or several items from media assets. Since ``CastPlayer`` conforms to [`ObservableObject`](https://developer.apple.com/documentation/combine/observableobject) and cannot be instantiated manually, it should be stored as an [`ObservedObject`](https://developer.apple.com/documentation/swiftui/observedobject) within a SwiftUI view. This ensures the player's lifecycle aligns with the view and that UI updates automatically reflect changes in playback state.
```